lineax/_solver/qr.py

```diff
@@ -79,7 +79,7 @@ def compute(
         else:
             # Least squares solution if overdetermined.
             solution = jsp.linalg.solve_triangular(
-                r, q.T @ vector, trans="N", unit_diagonal=False
+                r, q.T.conj() @ vector, trans="N", unit_diagonal=False
```

Comment:
I suspect there should be a `conj` somewhere on the `if transpose` branch as well.